timemachine/fe/bar.py

```diff
@@ -179,12 +179,8 @@ def bootstrap_bar(
     * TODO[deboggle] -- upgrade from pymbar3 to pymbar4 and remove this
     * TODO[performance] -- multiprocessing, if needed?
     """
-    u_kn, N_k = ukln_to_ukn(u_kln)
-    mbar = pymbar.MBAR(u_kn, N_k, maximum_iterations=maximum_iterations)
 
-    df, ddf = mbar.getFreeEnergyDifferences()
-    full_bar_result = df[0, 1]
-    full_bar_err = ddf[0, 1]
+    full_bar_result, full_bar_err = df_and_err_from_u_kln(u_kln, maximum_iterations=maximum_iterations)
```

Comment:
df_and_err_from_u_kln handles the behavior correctly: https://github.com/proteneer/timemachine/blob/master/timemachine/fe/bar.py#L136-L139